raiden/tests/scenarios/pfs5_too_low_capacity.yaml

```diff
@@ -54,6 +54,7 @@ scenario:
           name: "Test shortest path route"
           tasks:
             # Check that a payment is made with shortest path [0, 4, 3] from 0 to 3
+            - wait_blocks: 2
```

Comment:
Not sure why this would fix anything. You're basically waiting before triggering a transfer which gains no benefit IMO.

Reply:
I think that the pfs does not have the correct channel state immediate after channel opening. It must be udated with the PFSUpdate-Messages. It might happen that sometimes this messages has some latency, that might make the test flaky. I can’t see the node logs now, will hopefully get them tomorrow to see. But for tonights run, that might fix it. We had a similliar problem at another scenario

Reply:
the pfs logs show 
```
ABLE: 2>, '0xb89943BDF1AC3C66450b0cef9B4F232754b68892': <AddressReachability.UNREACHABLE: 2>, '0xaCC182dD70acDcd10635eC026Adf106E4ebab904': <AddressReachability.UNREACHABLE: 2>, **'0xd7E1533c68Bd9c992155898D20694658297855f4'**: <AddressReachability.REACHABLE: 1>, '0xF2204F44b62208BE92135555de0Ab2b6627825d7': <AddressReachability.REACHABLE: 1>, '0x917FF8B908bFdc20BEb8D0A41544F6D6b27BB299': <AddressReachability.REACHABLE: 1>, '0x1B3bD9776a610513AD6afCA6F670e7d9BB5C87Dc': <AddressReachability.REACHABLE: 1>, '0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee': <AddressReachability.REACHABLE: 1>} source=0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee target=0xF2204F44b62208BE92135555de0Ab2b6627825d7 value=1000000000000000
```

so node 4 (0xd7E1533c68Bd9c992155898D20694658297855f4) was reachable, but the path was considered invalid because of missing capacity 

``` 
2019-10-06 02:30:46.331071 [debug    ] Checking path validity         [pathfinding_service.model.token_network] nodes=['0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee', '0xd7E1533c68Bd9c992155898D20694658297855f4', '0xF2204F44b62208BE92135555de0Ab2b6627825d7'] value=1000000000000000
2019-10-06 02:30:46.333855 [debug    ] Path invalid because of missing capacity [pathfinding_service.model.token_network] available_capacity=0 edge={'view': <ChannelView cid=1263 from=0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee to=0xd7E1533c68Bd9c992155898D20694658297855f4 capacity=0>} fee=[0, 0] required_capacity=1000000000000000
2019-10-06 02:30:46.339045 [debug    ] Creating Path object           [pathfinding_service.model.token_network] fees=[0] is_valid=False nodes=['0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee', '0xd7E1533c68Bd9c992155898D20694658297855f4', '0xF2204F44b62208BE92135555de0Ab2b6627825d7']
2019-10-06 02:30:46.342316 [debug    ] Checking path validity         [pathfinding_service.model.token_network] nodes=['0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee', '0xd7E1533c68Bd9c992155898D20694658297855f4', '0xF2204F44b62208BE92135555de0Ab2b6627825d7'] value=1000000000000000
2019-10-06 02:30:46.343867 [debug    ] Path invalid because of missing capacity [pathfinding_service.model.token_network] available_capacity=0 edge={'view': <ChannelView cid=1263 from=0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee to=0xd7E1533c68Bd9c992155898D20694658297855f4 capacity=0>} fee=[0, 0] required_capacity=1000000000000000
172.18.0.21 - - [2019-10-06 02:30:46] "POST /api/v1/0x9526EC0E58ac5Bb1e24d655E964f0eAA46054d16/paths HTTP/1.1" 200 532 0.184877
```

... and then the Capacity Update reached the PFS

```
2019-10-06 02:30:48.924697 [debug    ] Received Capacity Update       [pathfinding_service.service] message={'signature': '0xd90af19734990fe455c0171dd805399dd2f50a23c2667f5f8465d37ba76d672c6fa93747d5ae9aca7c0eb84859302475dbcb713e748f42ca282dcc1e5f830d291c', 'canonical_identifier': {'chain_identifier': 5, 'token_network_address': '0x9526EC0E58ac5Bb1e24d655E964f0eAA46054d16', 'channel_identifier': 1263}, 'updating_participant': '0xd7E1533c68Bd9c992155898D20694658297855f4', 'other_participant': '0x3d37cB2f64f325d1a87943C30d4f9BB4C104D9ee', 'updating_nonce': 0, 'other_nonce': 0, 'updating_capacity': 0, 'other_capacity': 1000000000000000000, 'reveal_timeout': 50, 'type_name': 'PFSCapacityUpdate'}

``` 

so we need the wait before the transfer and after the channel openings